crates/iceberg-catalog/src/api/router.rs

```diff
@@ -40,11 +43,16 @@ pub fn new_full_router<
         State<A, C, S>,
     >();
     let management_routes = Router::new().merge(ApiServer::new_v1_router());
+
     maybe_add_auth(
         token_verifier,
         Router::new()
             .nest("/catalog/v1", v1_routes)
-            .nest("/management/v1", management_routes),
+            .nest("/management/v1", management_routes)
+            .merge(utoipa_swagger_ui::SwaggerUi::new("/swagger-ui").url(
```

Comment:
i think this puts swagger-ui under auth, do we want that?

Reply:
adding it outside of `maybe_add_auth` should make swagger & openapi json available without auth

Reply:
Done